doc/source/serve/tutorials/gradio-integration.md

```diff
@@ -37,91 +37,91 @@ In order to deploy your Gradio app onto Ray Serve, you need to wrap your Gradio
 
 By wrapping your application in `GradioServer`, you can increase the number of CPUs and/or GPUs available to the application.
 :::{note}
-Currently, there is no support for routing requests properly to multiple replicas of `GradioServer`, so we recommend only having a single replica.
+Ray Serve doesn't support routing requests properly to multiple replicas of `GradioServer`, so you should only have a single replica.
 :::
 
 :::{note} 
-`GradioServer` is simply `GradioIngress` but wrapped in a Serve deployment. You can use `GradioServer` for the simple wrap-and-deploy use case, but as you will see in the next section, you can use `GradioIngress` to define your own Gradio Server for more customized use cases.
+`GradioServer` is simply `GradioIngress` but wrapped in a Serve deployment. You can use `GradioServer` for the simple wrap-and-deploy use case, but in the next section, you can use `GradioIngress` to define your own Gradio Server for more customized use cases.
 :::
 
 :::{note} 
 Ray can’t pickle Gradio. Instead, pass a builder function that constructs the Gradio interface.
 :::
 
-Using either Gradio app `io` constructed by the builder function above or providing your own application (of type `Interface`, `Block`, `Parallel`, etc.), wrap it in your Gradio Server. Pass the builder function as input to your Gradio Server. It will be used to construct your Gradio app on the Ray cluster.
+Using either Gradio app `io` constructed by the builder function above or providing your own application of type `Interface`, `Block`, `Parallel`, etc., wrap the app in your Gradio Server. Pass the builder function as input to your Gradio Server. Ray Serves uses the builder function to construct your Gradio app on the Ray cluster.
```

Comment:
@shrekris-anyscale: Do we mean, "Using either Gradio app `io`... or your own application", or
"Either use Gradio app `io` constructed by the builder or provide your own application"?

Reply:
I believe we mean "using either the Gradio app `io` (which is constructed by the builder) or your own Gradio application..."

Reply:
Ty!